components/collector/src/source_collectors/jira/issue_status.py

```diff
@@ -22,4 +24,6 @@ async def _parse_issue_status(self, responses: SourceResponses) -> IssueStatus:
         """Override to get the issue status from the responses."""
         json = await responses[0].json()
         name = json["fields"]["status"]["name"]
-        return IssueStatus(self._issue_id, name=name)
+        created = datetime.strptime(json["fields"]["created"], "%Y-%m-%dT%H:%M:%S.%f%z")
+        date_string = created.strftime("%d-%m-%Y")
```

Comment:
I think it's better to store the data in ISO-format and let the frontend deal with localization.